src/components/util/forms/form-table/editable-cell.jsx

```diff
@@ -0,0 +1,46 @@
+import { React, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Create an editable cell renderer This file is highly inspired, if not copy
```

Comment:
This file is built on https://github.com/tannerlinsley/react-table/tree/master/examples/editable-data